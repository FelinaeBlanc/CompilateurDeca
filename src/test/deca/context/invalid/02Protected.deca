class A {
    protected int slt = 12; 

}

{
    A a = new A() ;
    println(a.slt);

    if (!(a.slt || true) ){
        println("true"); 
    }
    else{
        println("false"); 
    }
}
