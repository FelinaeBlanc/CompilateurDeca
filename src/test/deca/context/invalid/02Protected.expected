Can't access protected property in main
